app/src/components/About.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from "react";
+import Header from "./Header";
+import Footer from "./Footer";
+import MainNav from "./shared/MainNav";
+import { useTranslation } from "react-i18next";
+import { Link, useLocation } from "react-router-dom";
+import cn from "classnames";
+import axios from 'axios';
+import i18n from "../i18n/i18n";
+import DOMPurify from "dompurify";
+
+const About: React.FC = () => {
+	const { t } = useTranslation();
+	const location = useLocation();
+
+	const [displayNavigation, setNavigation] = useState(false);
+	const [aboutContent, setAboutContent] = useState<string>("");
+
+	const toggleNavigation = () => {
+		setNavigation(!displayNavigation);
+	};
+
+	useEffect(() => {
+		const getURL = (language: string) => {
+			return `ui/config/admin-ui/${location.pathname.split("/").pop()}.${language}.html`;
+		};
+
+		axios.get(getURL(i18n.language))
+			.then(response => {
+				setAboutContent(response.data);
+			})
+			.catch(error => {
+				axios.get(getURL(typeof i18n.options.fallbackLng === 'string' ? i18n.options.fallbackLng : 'en-GB'))
+					.then(response => {
+						setAboutContent(response.data);
+					})
+					.catch(error => {
+						console.error('Error while fetching data:', error);
+						setAboutContent(t("ABOUT.NOCONTENT").toString());
+					});
+			});
+
+	}, [location.pathname]); // Listen to changes in pathname
```

Comment:
`eslint` complains here:
```
React Hook useEffect has a missing dependency: 't'.
```
This warning is "fixed" this way in other places in the code the same way.

```suggestion
           // eslint-disable-next-line react-hooks/exhaustive-deps
	}, [location.pathname]); // Listen to changes in pathname
```